packages/zstd.rb

```diff
@@ -0,0 +1,17 @@
+require 'package'
+
+class Zstd < Package
+  description 'Zstandard - Fast real-time compression algorithm'
+  homepage 'http://www.zstd.net'
+  version '1.3.4'
+  source_url 'https://github.com/facebook/zstd/archive/master.tar.gz'
+  source_sha256 '7fba0070789a3c1e343fab4660958b73cc262110d2977589ba158b169d331ad6'
+  
+  def self.build
+    system 'make'
```

Comment:
Is there a configure script?  If so, we will need the prefix and possibly the libdir abstracted.